app/page/template/content/conversation/message-list.htm

```diff
@@ -9,7 +9,7 @@
                       attr: {'data-uie-uid': message.id, 'data-uie-value': message.super_type, 'data-uie-expired-status': message.ephemeral_expires, 'data-uie-send-status': message.status}"
            data-uie-name="item-message">
         <div data-bind="template: {name: 'timestamp'}"></div>
-        <div data-bind="template: {name: message.super_type}, css: {'message-isreplying': message.isReplying()}"></div>
+        <div data-bind="template: {name: message.super_type}, css: {'message-isreplying': message.id === $root.inputBar.replyMessageId()}"></div>
```

Comment:
As agreed with @atomrc, we need to get rid of observables on the message entity which are reading states from the input bar.